src/lib/transform.js

```diff
@@ -82,6 +85,11 @@ function makeGoalsAndObjectivesObject(objectiveRecords) {
       objectiveNum = 1;
     }
 
+    // goal number should be at least 1
+    if (!goalNum) {
+      goalNum = 1;
+    }
+
```

Comment:
What do you think about making the same check for the objectives? Right now I am seeing Objective 1.0 for non-grantees. (unless the was done on purpose).

Reply:
This is a good point, I was thinking it made sense the way I had it, however I realize that it will create other columns (like objective-1.0) that don't really make sense with the way the rest of them are named.